gen3/bin/approve_vpcpeering_request.sh

```diff
@@ -1,23 +1,45 @@
 #!/bin/bash
-if [ $# -ne 1 ]; then
+if [ $# -lt 1 ]; then
     echo "USAGE: $0 name_of_child_vpc"
```

Comment:
Would be helpful to update the Usage and/or add comments on the new flag that's being introduced and how to use it

Reply:
Ya sure. I'll add an readme for it.